test/Models/WebMapServiceCatalogGroupSpec.js

```diff
@@ -77,14 +78,19 @@ describe('WebMapServiceCatalogGroup', function() {
         group.flatten = true;
 
         loadWithXhr.load = function(url, responseType, method, data, headers, deferred, overrideMimeType, preferText) {
-            expect(url).toContain('GetCapabilities');
-            return loadWithXhr.defaultLoad('test/GetCapabilities/GeographicOnly.xml', responseType, method, data, headers, deferred, overrideMimeType, preferText);
+            if (url.indexOf('GetCapabilities') >= 0) {
```

Comment:
Only because all the other tests in here already used this technique.  Let me see if I can easily switch them all over.